app/controllers/api/functions.php

```diff
@@ -183,7 +183,13 @@
     ->param('templateVersion', '', new Text(128, 0), 'Version (tag) for the repo linked to the function template.', true)
     ->param('specification', APP_FUNCTION_SPECIFICATION_DEFAULT, fn (array $plan) => new RuntimeSpecification(
         $plan,
-        Config::getParam('runtime-specifications', []),
+        Config::getParam('runtime-specifications', [
+            APP_FUNCTION_SPECIFICATION_DEFAULT => [
+                'slug ' => APP_FUNCTION_SPECIFICATION_DEFAULT,
+                'memory' => APP_FUNCTION_MEMORY_DEFAULT,
+                'cpus' => APP_FUNCTION_CPUS_DEFAULT
+            ]
+        ]),
```

Comment:
I think we should fully remove specification from open source, and only relay on values in env variables (for cpus and memory).
That will also mean removing params for it, removing it from collections, response filters, ...